src/abhishekSJun20/UnionOfArray.java

```diff
@@ -0,0 +1,81 @@
+package abhishekSJun20;
+
+public class UnionOfArray {
+	int count = 0;
+	// method will find out unique elements from Two given array and forming a union of these two in a third array.
+	void getUnionOfArray(double[] arr1, double[] arr2) {
+		boolean flag = true;
+		int arrlength = arr1.length + arr2.length;
+		double[] unionOfTwoArr = new double[arrlength];
+		// first take the unique elements of arr1 insert in resultant union of array.
+		for (int outerindex = 0; outerindex < arr1.length; outerindex++) {
+			for (int innerindex = 0; innerindex <= count; innerindex++) {
+				if (unionOfTwoArr[innerindex] != arr1[outerindex]) {
+					flag = isNumberUnique(unionOfTwoArr, arr1[outerindex]);
+					if (!flag) {
+						unionOfTwoArr[count] = arr1[outerindex];
+						count++;
+						break;
+					}
+				}
+			}
+		}
+		// Take unique elements from second array arr2 and insert in resultant union of array
+		for (int outerindex = 0; outerindex < arr2.length; outerindex++) {
+			for (int innerindex = count; innerindex <= count; innerindex++) {
+				if (unionOfTwoArr[innerindex] != arr2[outerindex]) {
+					flag = isNumberUnique(unionOfTwoArr, arr2[outerindex]);
```

Comment:
isNumberUnique(), no need of this method. without it you can perform the operations. Remove it.